src/Widgets/HeaderBar.vala

```diff
@@ -54,21 +59,27 @@ public class Camera.Widgets.HeaderBar : Gtk.HeaderBar {
     }
 
     construct {
+        timer_button = new Widgets.TimerButton ();
+
+        timer_revealer = new Gtk.Revealer ();
```

Comment:
Instead of hiding this while video is selected, it would probably be better to make it insensitive